### DISCUSSION TITLE
How does a DAO work? Excellent primer, must watch!

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1131] [General]
Seeing DAOs popping up left and right, I asked myself, how to they deal with voter fraud? I just watched Patrick Collins new youtube article [What is a DAO? How to Build a DAO? (High Level)](https://youtu.be/X_QKZzd68ro)
This helped me a lot to understand the challenges a DAO faces when it comes to making decisions.

**Comment by cromewar** [comment]
One way is to restrict the amount of times each wallet cant vote and set a minimum holding amount in order to be able to vote, is effective but some would say is not decentralized. 
Honestly I don't know how to solve this problems, may be this is an opportunity to create a protocol which solves this....

**Reply by n4n0b1t3** [reply]
https://vitalik.ca/general/2022/01/26/soulbound.html
That would be something.
